<launch> 

    <!-- start approach person service -->
    <node pkg="pioneer_shr" type="approach_person_service" name="approach_person_service" output="screen">
		<param name="predefined1PositionX" type="double" value="2.6008"/>
		<param name="predefined1PositionY" type="double" value="-0.4473"/>
		<param name="predefined1OrientationZ" type="double" value="0.8224"/>
		<param name="predefined1OrientationW" type="double" value="0.5687"/>
		<param name="predefined1Name" type="string" value="SPOT_A"/>
        <param name="predefined2PositionX" type="double" value="2.5491"/>
		<param name="predefined2PositionY" type="double" value="3.0393"/>
		<param name="predefined2OrientationZ" type="double" value="0.8401"/>
		<param name="predefined2OrientationW" type="double" value="0.5423"/>
		<param name="predefined2Name" type="string" value="SPOT_B"/>
		<param name="targetPersonName" type="string" value="Alex"/>
		<param name="cameraFrameId" type="string" value="camera_link_optical"/>
	</node>

    <!-- start run script service -->
    <node pkg="pioneer_shr" type="run_script_service" name="run_script_service" output="screen">
    </node>

    <!-- start DB monitor service -->
    <node pkg="pioneer_shr" type="db_monitor_service" name="db_monitor_service" output="screen">
    </node>

    <!-- start move to service -->
    <node pkg="pioneer_shr" type="move_to_service" name="move_to_service" output="screen">
		<param name="predefined1PositionX" type="double" value="2.6008"/>
		<param name="predefined1PositionY" type="double" value="-0.4473"/>
		<param name="predefined1OrientationZ" type="double" value="0.8224"/>
		<param name="predefined1OrientationW" type="double" value="0.5687"/>
		<param name="predefined1Name" type="string" value="SPOT_A"/>
        <param name="predefined2PositionX" type="double" value="2.5491"/>
		<param name="predefined2PositionY" type="double" value="3.0393"/>
		<param name="predefined2OrientationZ" type="double" value="0.8401"/>
		<param name="predefined2OrientationW" type="double" value="0.5423"/>
		<param name="predefined2Name" type="string" value="SPOT_B"/>
        <param name="predefined3PositionX" type="double" value="2.5491"/>
		<param name="predefined3PositionY" type="double" value="3.0393"/>
		<param name="predefined3OrientationZ" type="double" value="0.8401"/>
		<param name="predefined3OrientationW" type="double" value="0.5423"/>
		<param name="predefined3Name" type="string" value="door"/>
	</node>


</launch>
